noise2same/network.py

```diff
@@ -526,6 +545,12 @@ def __init__(self, unit, in_channels, out_channels, global_ratio):
             in_channels=in_channels_global, out_channels=out_channels_global
         )  # global
 
+    def forward(self, x: T):
+        x_l, x_g = x
```

Comment:
You could add an `assert isinstance(x, tuple)` just in case. Or assign an empty tensor to `x_g` (is it how it's done in FFC?)